benchmark/README.md

```diff
@@ -14,9 +14,28 @@ This folder contains benchmarks to measure the performance of the Node.js APIs.
 
 ## Prerequisites
 
-Most of the http benchmarks require [`wrk`][wrk] to be installed. It may be
-available through your preferred package manager. If not, `wrk` can be built
-[from source][wrk] via `make`.
+Most of the HTTP benchmarks require a benchmarker to be installed, this can be
+either [`wrk`][wrk] or [`autocannon`][autocannon].
+
+`Autocannon` is a Node script that can be installed using
+`npm install -g autocannon`. It will use the Node executable that is in the
+path, hence if you want to compare two HTTP benchmark runs make sure that the
+Node version in the path is not altered.
+
+`wrk` may be available through your preferred package manger. If not, you can
+easily build it [from source][wrk] via `make`.
+
+To select which tool will be used to run your HTTP benchmark you can:
+* When running the benchmakrs, set `NODE_HTTP_BENCHMARKER` environment variable
+to desired benchmarker.
+* To select the default benchmarker for a particular benchmark, specify it as
+`benchmarker` key (e.g. `benchmarker: 'wrk'`) in configuration passed to
```

Comment:
I don't think this is a good idea. The options to `createBenchmark ` should just contain be benchmark parameters. If you want this feature, I think it would be much better to add an option object to `bench.http`. 

```js
var bench = common.createBenchmark(main, {
  num: [1, 4, 8, 16],
  size: [1, 64, 256],
  c: [100],
  benchmarker: ['wrk']
});

function main(conf) {
  bench.http({
    url: '/', 
    duration: 10,
    connections: conf.c,
    benchmarker: conf.benchmarker
  }, function () { ... });
}
```